src/db/connect.ts

```diff
@@ -1,14 +1,28 @@
-import {createConnection, Connection} from "typeorm";
+import { createConnection, Connection, ConnectionOptions } from "typeorm";
 import { ShortCode } from "./entities/shortcode.entity";
 
-export const connect = async () => await createConnection({
-    type: "postgres",
-    username: "sclrac",
-    password: "sclrac",
-    database: "sclrac",
-    logging: "all",
-    logger: "advanced-console",
-    entities: [ ShortCode ],
-    synchronize: true,
-    // dropSchema: true, // TODO: never in production, drops tables on restart
-});
\ No newline at end of file
+export const connect = async () => {
+    let options: ConnectionOptions = {
+        type: "postgres",
+        username: "sclrac",
+        password: "sclrac",
+        database: "sclrac",
+        logging: "all",
+        logger: "advanced-console",
+        entities: [ShortCode],
+        synchronize: true,
+    };
+    
+    if (process.env.NODE_ENV == 'production') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/scaleracademy/sclr.ac-backend/pullRequest?prid=7702321)
```suggestion
    if (process.env.NODE_ENV == "production") {
```